www/docs/server/merging-routers.md

```diff
@@ -99,6 +99,85 @@ const appRouter = router({
 });
 ```
 
+:::info
+
+We recommend you to only define inline sub-routers within a file, and to keep the exported routers as a `t.router` object. This makes any potential type errors show up in the file they originate from, and not at the place where you merge them.
```

Comment:
```suggestion
We recommend you always export routers via `t.router`, as opposed to exporting objects with procedures attached. This ensures that type errors show up in the file they originate from, instead of the place where you merge them.
```

Reply:
This line was confusing me a tad, just a stab at clarifying it